libminifi/include/utils/HTTPClient.h

```diff
@@ -66,37 +69,89 @@ struct HTTPReadCallback {
   }
 };
 
+enum class SSLVersion : uint8_t {
+  SSLv2,
+  SSLv3,
+  TLSv1_0,
+  TLSv1_1,
+  TLSv1_2,
+};
+
 struct HTTPHeaderResponse {
  public:
-
   HTTPHeaderResponse(int max)
-      : max_tokens_(max) {
+   : max_tokens_(max)
+   , parsed(false) {
   }
 
-  void append(const std::string &header) {
+  DEPRECATED void append(const std::string &header) {
```

Comment:
Can  you add a comment on when this was deprecated so that we can do it in a timely manner since we are coming to the close of 0.x we should probably remove this after 1.0.

Reply:
Sure, will do.